christfellowshipchurch/web-app-v2
# Adjust Radio Buttons on Sign in and Edit Profile

### About
Adjusted the placement of the Radio buttons on Sign in and Edit Profile

### Test Instructions
Edit your profile and you should see that `Allow email notifications` and `Allow text notifications` are now aligned with the radio button horizontally.

Open the sign-in modal and you should see that `female` and `male` are now aligned with the radio button horizontally.

### Screenshots
<img width="853" alt="Screen Shot 2023-06-06 at 4 39 30 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/46769629/6a92bfe7-7dcb-479d-b9f7-a7591b12af0c">

<img width="750" alt="Screen Shot 2023-06-06 at 4 42 12 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/46769629/f86bc9f8-b407-4c63-bed0-b5022b6e2900">


### Closes Tickets
[[CFDP-2597]](https://christfellowshipchurch.atlassian.net/jira/software/projects/CFDP/boards/2?selectedIssue=CFDP-2597)

### Related Tickets
N/A




[CFDP-2597]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2597?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
